client/src/App.js

```diff
@@ -62,24 +36,15 @@ class App extends Component {
       );
     } else {
       return (
-        <div>
-          <Navbar action={this.changeTab} />
-          <div>
-            {this.state.currentTab === "Dashboard" ? (
-              <Dashboard />
-            ) : this.state.currentTab === "Create TimeTable" ? (
-              <CreateTimeTable />
-            ) : this.state.currentTab === "Share TimeTable" ? (
-              <ShareTimeTable />
-            ) : this.state.currentTab === "About Us" ? (
-              <AboutUs />
-            ) : this.state.currentTab === "Logout" ? (
-              <Landing></Landing>
-            ) : (
-              <h1 align='center'>"Page not available"</h1>
-            )}
-          </div>
-        </div>
+        <Provider store={store}>
+          <BrowserRouter>
+            <Navbar action={this.logout} />
+            <Route exact path="/" component={Dashboard} />
```

Comment:
This needs to have its separate route `/Dashboard` imo

Reply:
hmm